packages/styled-components/src/components/primitives/Chat/Conversation/Message/Message.styled.ts

```diff
@@ -95,42 +92,37 @@ export const StyledMediaWrapper = styled.div<Props>`
   .imageWrapper {
     display: flex;
   }
+`;
 
-  .mediaPreview {
+export const mediaPreview = `
```

Comment:
Only styled elements belong to this file

Reply:
Decided to let the that style in .styled.ts file.

Reply:
I like it in here but lets use a _better name_, `mediaPreviewStyling` or something like that